forager/_utils/config.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 from typing import Tuple, TypeVar
 
-def to_tuple(i: int | Tuple[int, int]):
+def to_tuple(i: int | Tuple[int, int] | None):
```

Comment:
I think we shouldn't place the fix here, actually. This `to_tuple` method should take some data and always produce a tuple. With this change, it might produce either a tuple or a None.

Reply:
How would you resolve this? By adding None checking?

Reply:
I'd probably fix it at the call-site of this function, [here](https://github.com/andnp/forager/pull/13/files#diff-c1a915a2b95cff934c45808be74c793ebfb22de9f3d2ed19889bc212995a253aL31)

Something like:
```python
self._ap_size = cu.to_tuple(config.aperture) if config.aperture is not None else None
```